apps/files_sharing/src/components/SharingEntry.vue

```diff
@@ -218,9 +260,11 @@ export default {
 	},
 
 	methods: {
-		updatePermissions(isEditChecked, isReshareChecked) {
+		updatePermissions(isEditChecked, isCreateChecked, isDeleteChecked, isReshareChecked) {
```

Comment:
```suggestion
		updatePermissions({ isEditChecked = this.canEdit, isCreateChecked = this.canCreate, isDeleteChecked = this.canDelete, isReshareChecked = this.canReshare } = {}) {
```

I would do something like that.
That way you can just execute the function while only providing one parameter.


```js
this.updatePermissions({ isDeleteChecked: checked })
```